### ISSUE TITLE
Python 3.11 Compatibility

### ISSUE THREAD

**Issue by bptarpley** [Princeton-CDH/viapy#3]
When attempting to use this package in Python 3.11, an error from the attrdict dependency is thrown:

```
File "/usr/local/lib/python3.11/site-packages/viapy/api.py", line 4, in <module>
     from attrdict import AttrMap
   File "/usr/local/lib/python3.11/site-packages/attrdict/__init__.py", line 5, in <module>
     from attrdict.mapping import AttrMap
   File "/usr/local/lib/python3.11/site-packages/attrdict/mapping.py", line 4, in <module>
     from collections import Mapping
 ImportError: cannot import name 'Mapping' from 'collections' (/usr/local/lib/python3.11/collections/__init__.py)
```

It seems like this could be easily fixed by replacing the attrdict==2.0.0 requirement with, say, attrdict3==2.0.2

This should work because the attrdict3 package correctly imports "Mapping" from collections.abc:

https://github.com/pirofti/AttrDict3/blob/master/attrdict/mapping.py#L5

**Comment by rlskoeser** [comment]
Thank you for alerting me to the existence of attrdict3! I've been aware of the attrdict problem but thought I needed to migrate to addict.

 I'm finally working on some maintenance updates, hope to release a new version of viapy fairly soon.

**Comment by bptarpley** [comment]
Wonderful! This is a great library and I look forward to integrating it again once it's 3.11 compatible :)

**Comment by rlskoeser** [comment]
Just released v0.3 of viapy which has been updated for newer versions of python and django, including python 3.11.

Hopefully it works for your needs. Would love to hear what you're using it for!
